scripts/main.js

```diff
@@ -294,7 +294,7 @@ function check_movable(coordleft,coordtop)
 			if(wallsplit[4] && wallsplit[4] == 'teleport')
 			{
 				stop_movement();
-				if(window.confirm('Walk to next area?'))
+				if(confirm("Walk to next area?"))

```

Comment:
Please double check that just using confirm instead of window.confirm works as intended on the Nintendo 3DS Browser.

Reply:
I was checking to see the functionality without the window.confirm and decided that in some cases you wouldn't want it automatically going to the next area. While I was putting back the code, I forgot to put window.confirm(). Sorry!

Reply:
No problem. Just change it back to window.confirm, just to be sure